prac_06/guitars.py

```diff
@@ -0,0 +1,28 @@
+"""
+Estimate:  30 minutes
+Actual:    22 minutes
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Format user input and determine if guitar is vintage."""
+    print("My guitars!")
+    user_guitars = []
+    guitar_name = input("Name: ")
+    while guitar_name != "":
+        guitar_year = input("Year: ")
```

Comment:
Could possibly shorten these names to not include the word guitar as it is clear in this program that the name, cost and year are referring to a guitar attribute.

Reply:
Sorry for that late reply, thanks for the review 👍